src/types/RouteOptionsType.ts

```diff
@@ -1,4 +1,12 @@
 /* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
 /* SPDX-License-Identifier: MIT-0 */
 
-export type RouteOptionsType = { avoidTolls: boolean; avoidFerries: boolean };
+export type RouteOptionsType = {
+	avoidTolls: boolean;
+	avoidFerries: boolean;
+	avoidDirtRoads: boolean;
+	avoidUTurns: boolean;
+	avoidTunnels: boolean;
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	[key: string]: any; // Add index signature to allow string keys
```

Comment:
Is this required? If we need any other properties, lets define it on the type and avoid adding any generic string keys